src/gasman.c

```diff
@@ -1804,6 +1808,10 @@ UInt CollectBags (
     for ( i = 0; i < GlobalBags.nr; i++ )
         MARK_BAG( *GlobalBags.addr[i] );
 
+    /* allow users of libgap to mark their choice of bags */
+    if ( LibFuncBags ) {
+        (*LibFuncBags)();
```

Comment:
What is this supposed to be? An additional bag marking function, analog to `StackFuncBags`? I realize this is not chosen by you, but still, I feel compelled to remark this: This name is IMHO quite bad (worse than `StackFuncBags` which also strikes me as bad -- it should be called `MarkBagsOnStackFunc` or at least `MarkStackFuncsBags` (if one accepts the premise that all GASMAN functions and variables should end with the word "Bugs").

A better name might be `ExtraMarkFuncBags` or so...